src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -141,6 +173,20 @@ public uint GetStoragePrice(IReadOnlyStore snapshot)
             return (uint)(BigInteger)snapshot[_storagePrice];
         }
 
+        /// <summary>
+        /// Gets the block generation time in milliseconds.
+        /// This can only be called after the HF_Echidna.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <returns>The block generation time in milliseconds.</returns>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
+        public uint GetMSPerBlock(IReadOnlyStore snapshot)
+        {
+            // this should not be null since this method is only available after the HF_Echidna
+            snapshot.TryGet(_blockGenTime, out var item);
```

Comment:
If ! throw an exception instead of get null exception should be good for debug

Reply:
We should just use simple index expression here, like for any other Policy getter method. Will fix.

Reply:
Fixed.